vendor.mod

```diff
@@ -0,0 +1,197 @@
+	github.com/fsnotify/fsnotify v1.4.9
+	github.com/godbus/dbus/v5 v5.0.4
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/gddo v0.0.0-20190904175337-72a348e765d2
+	github.com/google/go-cmp v0.5.5
+	github.com/google/uuid v1.3.0
+	github.com/gorilla/mux v1.8.0
+	github.com/hashicorp/go-immutable-radix v1.0.0
+	github.com/hashicorp/go-memdb v0.0.0-20161216180745-cb9a474f84cc
+	github.com/hashicorp/memberlist v0.2.4
+	github.com/hashicorp/serf v0.8.2
+	github.com/imdario/mergo v0.3.12
+	github.com/ishidawataru/sctp v0.0.0-20210226210310-f2269e66cdee
+	github.com/klauspost/compress v1.12.3
+	github.com/miekg/dns v1.1.27
+	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
+	github.com/moby/buildkit v0.8.2-0.20210615162540-9f254e18360a
+	github.com/moby/ipvs v1.0.1
+	github.com/moby/locker v1.0.1
+	github.com/moby/sys/mount v0.3.0
+	github.com/moby/sys/mountinfo v0.5.0
+	github.com/moby/sys/signal v0.6.0
+	github.com/moby/sys/symlink v0.2.0
+	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
+	github.com/morikuni/aec v1.0.0
+	github.com/opencontainers/go-digest v1.0.0
+	github.com/opencontainers/image-spec v1.0.2
+	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/selinux v1.9.1
+	github.com/pelletier/go-toml v1.9.1
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.11.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/spf13/cobra v1.1.3
+	github.com/spf13/pflag v1.0.5
+	github.com/tchap/go-patricia v2.3.0+incompatible
+	github.com/tonistiigi/fsutil v0.0.0-20210609172227-d72af97c0eaf
+	github.com/vbatts/tar-split v0.11.2
+	github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852
+	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
+	go.etcd.io/bbolt v1.3.6
+	golang.org/x/net v0.0.0-20211112202133-69e39bad7dc2
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	golang.org/x/sys v0.0.0-20211025201205-69cdffdb9359
+	golang.org/x/time v0.0.0-20211116232009-f0f3c7e86c11
+	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa
+	google.golang.org/grpc v1.40.0
+	gotest.tools/v3 v3.0.3
+)
+
+require (
+	cloud.google.com/go/bigquery v0.0.0-00010101000000-000000000000 // indirect
+	code.cloudfoundry.org/clock v1.0.0 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
+	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/cilium/ebpf v0.6.2 // indirect
+	github.com/containerd/console v1.0.2 // indirect
+	github.com/containerd/go-runc v1.0.0 // indirect
+	github.com/containerd/ttrpc v1.1.0 // indirect
+	github.com/coreos/etcd v3.3.25+incompatible // indirect
```

Comment:
Note to self; I need to update this to the version that's used in vendor.conf (see https://github.com/moby/moby/pull/41791)